mobile/src/main/java/ch/epfl/sdp/mobile/application/tournaments/TournamentFacade.kt

```diff
@@ -11,6 +16,20 @@ import ch.epfl.sdp.mobile.infrastructure.persistence.store.Store
  * @param store the [Store] which is used to manage documents.
  */
 class TournamentFacade(private val auth: Auth, private val store: Store) {
+
+  /**
+   * Returns all of the registered tournaments of the application.
+   *
+   * @param user the current [AuthenticatedUser].
+   */
+  fun getTournaments(user: AuthenticatedUser): Flow<List<Tournament>> {
```

Comment:
I'd name this `tournaments(...)` to stay consistent with standard Kotlin naming.

Reply:
Fixed in 5d32a9fb42579af9f705b9889170f812091800b2